support-models/src/main/scala/com/gu/support/zuora/api/response/QueryResponse.scala

```diff
@@ -19,10 +27,22 @@ object PaymentMethodRecord {
   implicit val codec: Codec[PaymentMethodRecord] = deriveCodec
 }
 
+object SubscriptionRedemptionFields {
+  implicit val decoder: Decoder[SubscriptionRedemptionFields] = decapitalizingDecoder[SubscriptionRedemptionFields]
+    .prepare(_.withFocus(_.mapObject(_
+      .renameField("RedemptionCode__c", "RedemptionCode")
+      .renameField("GifteeIdentityId__c", "GifteeIdentityId")
```

Comment:
presume this does work ok with the upper case `GifteeIdentityId` rather than `gifteeIdentityId` as in the case class?  but should it match?

Reply:
nm just realised it's a decapitalizingDecoder